### DISCUSSION TITLE
Multiple date ranges with possibility of overlap

### DISCUSSION THREAD

**Discussion by sonofGod225** [primefaces/community#423] [PrimeNG] [answered]
Hello team Prime Is it possible to have several date ranges on the same calendar with the possibility of overlapping? like in the screenshot.

<img width="516" alt="Capture d’écran 2023-11-09 à 12 13 29" src="https://github.com/primefaces/community/assets/20479925/dc9dd1d9-2aa8-46de-85b3-bf8ab637b334">

**Comment by cetincakiroglu** [comment, marked as answer]
Hi, 

Unfortunately, calendar doesn't have such a feature.
